src/api/cpp/cvc5.h

```diff
@@ -4043,21 +4069,23 @@ class CVC5_EXPORT Solver
 
   /**
    * Get the value of the given term in the current model.
+   *
    * SMT-LIB:
-   * \verbatim
-   * ( get-value ( <term> ) )
-   * \endverbatim
+   *
+   *     ( get-value ( <term> ) )
+   *
    * @param term the term for which the value is queried
    * @return the value of the given term
    */
   Term getValue(const Term& term) const;
 
   /**
    * Get the values of the given terms in the current model.
+   *
    * SMT-LIB:
-   * \verbatim
-   * ( get-value ( <term>+ ) )
-   * \endverbatim
+   *
+   *     ( get-value ( <term>+ ) )
```

Comment:
Technically, this should be `<term>*`...

Reply:
According to the SMT-LIB standard,it is required that `n > 0` for terms `t1...tn`. We do not guard against this though, we simply return an empty vector in that case. So yeah, I guess we should update this to `*`.